CosmosApiMessagesChangeFeed/function.json

```diff
@@ -0,0 +1,19 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "cosmosDBTrigger",
+      "name": "cosmosApiMessagesTrigger",
+      "direction": "in",
+      "connectionStringSetting": "COSMOS_API_CONNECTION_STRING",
+      "databaseName": "%COSMOSDB_NAME%",
+      "collectionName": "messages",
+      "leaseCollectionPrefix": "%MESSAGES_LEASES_PREFIX%-",
+      "leaseCollectionName": "pdnd-leases",
+      "createLeaseCollectionIfNotExists": true,
+      "startFromBeginning": true
+
+    }
+  ],
```

Comment:
We could add a short retry configuration

Reply:
Thanks, done in [4397783](https://github.com/pagopa/io-functions-elt/pull/39/commits/439778311e70c740a37991382f0eb23dc3fdcfb0)